pages/index.tsx

```diff
@@ -193,19 +213,22 @@ const Home: NextPage = () => {
             {currentView === VIEWS.INITIAL_VIEW ? (
               <img src="/citydao-parcel-0-NFT-Art.png" alt="Parcel Zero NFT" />
             ) : (
-              <MintedNftsView name="sss" navigateToHome={navigateToHome} />
+              <MintedNftsView
+                numberOfNfts={numberOfMintedNfts}
+                navigateToHome={navigateToHome}
+              />
             )}
           </div>
           <div className="content-right">
             {/* nftCount */}
             <button
               disabled={!address}
               onClick={numberOfMintedNfts === 0 ? claim : showMintedNfts}
-              className={address ? "border-button default-cursor" : ""}
+              className={
```

Comment:
would be good to create a generic button component that handles disabled styling using the disabled flag